Dockerfile

```diff
@@ -0,0 +1,25 @@
+
+# BUILD BIN
+
+FROM golang:1.22.0 as builder
+# Install go modules
+WORKDIR /flow-evm-gateway
+COPY go.* ./
+COPY . ./
+RUN go mod download
+RUN go mod verify
+RUN CGO_ENABLED=0 go build -o evm-gateway ./cmd/server/main.go
+RUN chmod a+x evm-gateway
+RUN chmod a+x ./scripts/run.sh
+RUN git clone https://github.com/m-Peter/flow-cli-custom-builds.git
```

Comment:
Minor, I run into a glitch the first time building the image locally:
```bash
> [builder 10/10] RUN git clone https://github.com/m-Peter/flow-cli-custom-builds.git:
0.344 Cloning into 'flow-cli-custom-builds'...
66.27 error: 3872 bytes of body are still expected
66.28 fetch-pack: unexpected disconnect while reading sideband packet
66.29 fatal: early EOF
66.31 fatal: fetch-pack: invalid index-pack output
------
ERROR: failed to solve: executor failed running [/bin/sh -c git clone https://github.com/m-Peter/flow-cli-custom-builds.git]: exit code: 128
```

Maybe another alternative could be:
```Dockerfile
RUN wget https://github.com/m-Peter/flow-cli-custom-builds/raw/main/flow-x86_64-linux-
RUN chmod a+x flow-x86_64-linux-
```

With the 2nd attempt, I was able to build the image and run a container, so it is not a big deal.